sklearn/manifold/_mds.py

```diff
@@ -335,6 +335,16 @@ class MDS(BaseEstimator):
         The final value of the stress (sum of squared distance of the
         disparities and the distances for all constrained points).
 
+    dissimilarity_matrix_ : ndarray, shape (n_samples, n_samples)
+        Pairwise dissimilarities between the points. Symmetric matrix that
+        - either uses a custom dissimilarity matrix by setting dissimilarity
```

Comment:
The formatting looks weird: https://109037-843222-gh.circle-artifacts.com/0/doc/modules/generated/sklearn.manifold.MDS.html

I have to say I am not sure how to write the proper reST here ... probably looking at the contributor doc to generate the documentation locall ywould be a good idea.

Reply:
In principle a blank line before the bullet list would be enough.

Reply:
yes we will need a blank line.